dist/index.html

```diff
@@ -76,6 +77,15 @@ <h2 class="recipe-name">Your Recipe here</h2>
             </ul>
           </div>
         </header>
+        <aside class="grocery-list-and-cost">
+          All Saved Recipe Ingredients:
```

Comment:
how do you feel about titling it Grocery List instead to fit the prompt better?

Reply:
Hmmm... I have an article right below with that class name. I named the aside class that because I thought it made sense considering the two articles below, but I'm open to suggestions!

Reply:
I agree, with Calli on this one where the title name should maybe be "Grocery List". The class names also make sense in the way you wrote them out! I don't think we would need to change them. But this can be done in a refactor after we've had time to talk about it as a team.